class APIError(Exception):
    
    pass
class APIErrorKey(Exception):
    #print('Con esta clave no hay acceso')
    #input('Escrbe la clave de nuevo')
    conectado=False
    pass

